source/geod24/LocalRest.d

```diff
@@ -553,31 +711,78 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
             // Control the node behavior
             Control control;
 
+            struct AwaitingMessages
```

Comment:
AwaitingMessages => AwaitingMessage

It's only a single message, and `await_msgs` is the array.